firmware/src/sys/board.cpp

```diff
@@ -497,18 +497,14 @@ unsigned getSupplyVoltageInMillivolts()     //error under 2%
 
 unsigned getOutVoltageInVolts()
 {
-    static std::uint16_t old_value = 0;
-
-    std::uint16_t new_value = 0;
-    (void)Chip_ADC_ReadValue(LPC_ADC, ADC_CH0, &new_value);
-
+    std::uint16_t result = 0;
```

Comment:
Hmm...I tried a bunch of things but could not get averaging to work right,...so now it's gone. Tested on a few units, noise seems to be very low